plugins/module_utils/_vmware_rest_client.py

```diff
@@ -191,51 +193,54 @@ def get_vm_by_name(self, name):
 
         return vms[0]
 
-    def get_library_item_by_name(self, name):
+    def get_content_library_ids(self, name=None, library_type=None, fail_on_missing=False):
```

Comment:
Why did you change this behavior?

Reply:
I saw that the other functions are still by name
@machacekondra what do you think?

Reply:
@machacekondra @mikemorency I don't think so because they're all marked as private. If people use them and they break backward compatibility, that's their problem. That's what private means.

Apropos of nothing, it would be great if there would be some non-private module utils that I could re-use in `community.vmware`. Actually, I think this was the original plan :thinking:

Reply:
Yes we did make them private while we were still figuring out method naming and making changes. Sounds like we can stick with whats in the PR.

@mariolenz , Most of the pyvmomi/rest api base utils stay the same, but I still make the occasional functional change to the other utils. Ill make a ticket to review the function names and prep the pyvmomi/rest utils to make them public. Ill have to look at previous conversation to see if that can be done in a minor release or needs a major one